pkg/apis/pipeline/v1alpha1/resource_types.go

```diff
@@ -117,12 +117,12 @@ type PipelineResourceBinding struct {
 // PipelineResourceResult used to export the image name and digest as json
 type PipelineResourceResult struct {
 	// Name and Digest are deprecated.
-	Name   string `json:"name"`
-	Digest string `json:"digest"`
+	Name   string `json:"name,omitempty"`
+	Digest string `json:"digest,omitempty"`
```

Comment:
Since we're removing from imagedigestexporter, should we just remove these?  Have been deprecated since 0.8 https://github.com/tektoncd/pipeline/releases/tag/v0.8.0